unfccc.csl

```diff
@@ -0,0 +1,243 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" class="in-text" demote-non-dropping-particle="sort-only" default-locale="en-GB" page-range-format="expanded">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>UNFCCC</title>
+    <title-short>UNFCCC</title-short>
+    <id>http://www.zotero.org/styles/unfccc</id>
+    <link rel="self" href="http://www.zotero.org/styles/unfccc"/>
+    <link href="https://www.zotero.org/styles/harvard-kings-college-london" rel="template"/>
```

Comment:
As rmzelle says, we'll accept this with that link, but there's no way to make that public or attach it here? If someone complains that there's something wrong with the format, it's very helpful for us to be able to consult how it's supposed to look.